S/sox/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "sox"
+version = v"14.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://sourceforge.net/projects/sox/files/sox/$(version)/sox-$(version).tar.bz2",
+                  "81a6956d4330e75b5827316e44ae381e6f1e8928003c6aa45896da9041ea149c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/sox/
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/sox*
```

Reply:
@theogf thank you for trying. This should make it move on one step further at least